python/tk_desktop/notifications/python37_deprecation_notification.py

```diff
@@ -60,19 +54,19 @@ def message(self):
         Message to display.
         """
 
-        url = "https://community.shotgridsoftware.com/t/important-notice-for-end-of-february-2025-end-of-support-for-shotgrid-desktop-1-7-and-for-python-3-7-in-fptr-toolkit"
+        url = "https://community.shotgridsoftware.com/t/20515"
 
         add_fptr1, add_fptr2 = ("", "")
         if self.include_sgd:
-            add_fptr1 = "<b>ShotGrid Desktop 1.7</b> and"
-            add_fptr2 = "and <b>FPTR desktop 1.8</b>+"
+            add_fptr1 = " and <b>ShotGrid Desktop 1.7</b>"
+            add_fptr2 = " and <b>FPT Desktop v2.1.0</b>"
 
         return f"""
-            On <b>Feb 28th, 2025</b> Autodesk stops supporting {add_fptr1}
-            <b>Python 3.7</b> in FPTR Toolkit.
+            End of March 2026 - Breaking Changes for <b>Python 3.7</b>
+            {add_fptr1}
 
-            Update to <b>Python 3.9</b>+ {add_fptr2} before this date to avoid
-            disruption.
+            Make sure to update to <b>Python 3.11</b>{add_fptr2}, or any other
+            supported versions, before  April 1, 2026 to avoid disruption.
```

Comment:
```suggestion
            You must update to <b>Python 3.11</b>{add_fptr2}&mdash;or other
            supported versions&mdash;before April 1, 2026 to avoid disruption.
```
Two comments regarding this comment: trying `&mdash;` to see if it's allowed. Also, would be great for `or other supported versions` to be a link to documentation that lists the supported versions